Assets/Scripts/Internal/PlayerGPS.cs

```diff
@@ -426,6 +426,19 @@ public NameHelper.BankTypes GetNameBankOfCurrentRegion()
             return NameHelper.BankTypes.Breton;
         }
 
+        /// <summary>
+        /// Gets NameHelper.BankType in given region.
+        /// In practice this will always be Redguard/Breton.
+        /// Supporting other name banks for possible diversity later.
+        /// </summary>
+        public NameHelper.BankTypes GetNameBankOfRegion(int regionIndex)
```

Comment:
Why is this in PlayerGPS if it uses nothing about the player's current location? This just reaches into MapsFile based on the input. You can always put the helper in MapsFile

Reply:
Moved